roles/provision-crc-apb/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: Validate CRC JSON config
+  set_fact: svc_json_config={{ SERVICE_CONFIG | from_json }}
+  register: crc_json_config_valid
+  ignore_errors: True
+
+- fail:
```

Comment:
I think in this case we should use last_operation so the user is informed of the reason for the failure.

```
asb_last_operation:
    description: "config was not valid json. Could not provision"  
```

Reply:
We can use it instead of fail? Or we need to use it before fail?

Reply:
ok so I think a better solution (we should use last operation for each step happening) for the error case is to write the error to ```/dev/termination-log``` this is read by k8s and set as the termination status for the pod. This is checked by the broker and displayed if present